symfit/core/fit.py

```diff
@@ -1476,6 +1506,59 @@ def _init_constraints(self, constraints, model):
                     )
         return con_models
 
+    @staticmethod
+    def build_grad_lagrangian(objective, multipliers, constraint_objectives):
+        """
+        Build a :class:`~symfit.core.fit.CallableNumericalModel` representing
+        the gradient of the Lagrangian function.
+
+        The Lagrange function is
+
+        .. math:: \\mathcal{L}(\\vec{p}, \\vec{\\lambda}) = f(\\vec{p}) + \\sum_{j} \\lambda_j g_{j}(\\vec{p})
+
+        where :math:`f(\\vec{p})` is the objective function to be minimized as
+        a function of parameters :math:`\\vec{p}`, the :math:`\\lambda_j` are
+        the lagrange multipliers, and :math:`g_{j}(\\vec{p})` are the equality
+        constraints.
+
+        Because the constraints are met at stationary points of the Lagrange
+        function, we solve :math:`\\nabla_p \\mathcal{L} = 0` instead of
+        minimizing :math:`\\mathcal{L}` directly.
+
+        :param objective: Objective :math:`f(\\vec{p})`.
+        :param multipliers: list of :class:`symfit.core.argument.Parameter`
+            objects, to be interpreted as Lagrange multipliers :math:`\\lambda_j`.
+        :param constraint_objectives: list of objectives representing the
+            constraints :math:`g_{j}(\\vec{p})`.
```

Comment:
Why not just keep these two (multipliers, constraint_objectives) as dict for the argument here?